internal/logger/logger.go

```diff
@@ -76,21 +78,21 @@ func InitLogFile(filename string, format string) error {
 		sysWriter: sysWriter,
 		flag:      0,
 		format:    format,
+		level:     level,
 	}
-	defaultInfoLogger = NewInfo("")
-	defaultErrorLogger = NewError("")
+	defaultLogger = defaultLoggerFactory.newLogger(level)
 
 	return nil
 }
 
 // init initializes the logger factory to use stdout and stderr.
 func init() {
 	defaultLoggerFactory = &loggerFactory{
-		file: nil,
-		flag: log.Ldate | log.Ltime | log.Lmicroseconds,
+		file:  nil,
+		flag:  log.Ldate | log.Lmicroseconds,
```

Comment:
Why are we removing log.Ltime flag?

Reply:
These flags append date, time and microseconds to the log message. I noticed that log.Lmicroseconds already had time.
eg
```
//flag:      log.Ldate | log.Ltime | log.Lmicroseconds,
D0907 05:35:02.600719 fuse_debug: 05:35:02.600718 Op 0x00000150        connection.go:500] -> Error: "no such file or directory"

//flag:      log.Ldate | log.Ltime | log.Lmicroseconds,
D0907 05:32:16.375371 fuse_debug: 2023/09/07 05:32:16.375370 Op 0x00000150        connection.go:500] -> Error: "no such file or directory"
```

I see that we set this flag variable to 0 in initlogfile() method (removing the flags). As this is no longer being used, I will clean up this flag variable.